src/menu/index.tsx

```diff
@@ -1,18 +1,23 @@
 import React from 'react'
 import cc from 'classcat'
-import MenuItem, { MenuItemProps } from './menuItem'
-import style from './style'
+import ItemChoice, { ItemChoiceProps } from 'itemChoice'
 
 export interface MenuProps {
-  items: MenuItemProps[],
+  items: ItemChoiceProps[],
   className?: Classcat.Class,
 }
 
 const Menu = ({ className, items }: MenuProps) => (
-  <ul className={cc(className)} role="menu">
-    {items.map(item => <MenuItem {...item} key={item.id} />)}
-    <style jsx>{style}</style>
-  </ul>
+  <nav className={cc(className)} role="menu">
+    {items.map(item => (
+      <ItemChoice
```

Comment:
Does it use an li tag by default ? If it's the case you might need to keep the ul tag

Reply:
It's either a `<a>` if there's a `href` prop or a `<li>` otherwise

Reply:
If there's a `href` props that is a string, you get a `<a>`, yes